reel-picker/src/main/java/com/innowisegroup/reelpicker/picker/ReelPicker.kt

```diff
@@ -92,13 +92,6 @@ class ReelPicker<T> : DialogFragment() {
 
     private fun applyAttributes() {
         isCancelable = false
```

Comment:
useless function.
place this attr somewhere else in view initialization and remove function